server/events_test.go

```diff
@@ -813,6 +813,11 @@ func TestSystemAccountSystemConnectionLimitsHonored(t *testing.T) {
 		}
 		defer ncb1.Close()
 		tc++
+
+		// The account's connection count is exchanged between servers
+		// so that the local count on each server reflects the total count.
+		// Pause a bit to give a change for each server to process the update.
```

Comment:
chance

Reply:
also `for`->`to`: `give a chance to each server`.